<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Letter</class>
 <widget class="QWidget" name="Letter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Письмо {Letter}</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 12pt &quot;Verdana&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="VL">
     <item>
      <layout class="QHBoxLayout" name="SenderC">
       <item>
        <widget class="QLabel" name="FromWho">
         <property name="text">
          <string>От кого: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Who">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SubjectC">
       <item>
        <widget class="QLabel" name="SubjectIs">
         <property name="text">
          <string>Тема: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Subject">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="LetterContent">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
